hr_contract_currency/models/hr_contract.py

```diff
@@ -0,0 +1,28 @@
+# Copyright 2018 Brainbean Apps (https://brainbeanapps.com)
+# Copyright 2020 Onestein (<https://www.onestein.eu>)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+
+
+class HrContract(models.Model):
+    _inherit = "hr.contract"
+
+    currency_id = fields.Many2one(
+        "res.currency",
+        related=False,
+        readonly=False,
```

Comment:
By default related=False and readonly=False, It is not necessary to add it.

Reply:
It's an override, so necessary: https://github.com/odoo/odoo/blob/8f21148e1a8eaae3bc09ec54377becb2361e7834/addons/hr_contract/models/hr_contract.py#L79

Reply:
OK, Thank you for your suggest @alexey-pelykh

@aphon61bank Please back your code before.

Reply:
I have fixed it, Thank you.